src/services/user-settings/edit-excluded-categories.ts

```diff
@@ -0,0 +1,44 @@
+import UserSettings from '@models/UserSettings.model';
+import { withTransaction } from '../common';
+
+export const editExcludedCategories = withTransaction(
+  async ({
+    userId,
+    addIds = [],
+    removeIds = [],
+  }: {
+    userId: number;
+    addIds?: number[];
+    removeIds?: number[];
+  }): Promise<number[]> => {
+    const [existingSettings] = await UserSettings.findOrCreate({
+      where: { userId },
+      defaults: {
+        settings: { 
+          stats: {
+            expenses: {
+              excludedCategories: []
+            }
+          } 
+        },
```

Comment:
if you want to use defaults like that, they should be utilized somewhere with a type assigned, so if new settings will come, it will be updated and checked here as well

Reply:
How can i use defaults other ways?